### ISSUE TITLE
SAN not being included in the cert

### ISSUE THREAD

**Issue by titansmc** [broadinstitute/python-cert_manager#89]
Hi,
I am using this package to enroll SSL certs, so first I create a CSR (this I check that has all the SANs valid) and then I call ssl.enroll like:

`ssl.enroll(cert_type_name="OV Multi-Domain", csr=data, term=365, org_id=cert_org[0]["id"], external_requester="i@mail.com")`

This seems to work but the requested cert is not including any SAN, only the base domain ( like `domain.com`) and the www one (like `www.domain.com`)

**Comment by coreone** [comment]
I didn't see `subject_alt_names` being passed in that call to enroll. I believe that is necessary for the certificate to have any SANs attached, even if the CSR has SANs within it. If you enroll again for `domain.com` and pass `subject_alt_names="www.domain.com"`, does the certificate come out as expected?

**Comment by titansmc** [comment]
The following enroll call :
```
liste = ["log.domain.de"]

try:
    result = ssl.enroll(cert_type_name="OV Multi-Domain", csr=data, term=365, org_id=cert_org[0]["id"], subject_alt_names=liste)
```
would give me:
```
{'code': -1002, 'description': 'Cannot parse message body'}
Traceback (most recent call last):
  File "sectigo.py", line 48, in <module>
    cert_pem = ssl.collect(cert_id=result["sslId"], cert_format="x509CO")
NameError: name 'result' is not defined
```

**Comment by titansmc** [comment]
passing a string like this would work:

`liste = 'log.domain.de,sd.domain.de'`

**Comment by coreone** [comment]
@titansmc this should be fixed in #94
